### ISSUE TITLE
Versions import

### ISSUE THREAD

**Issue by globdesign** [GitHawkApp/GitHawk#2740]
How about adding a way to import a Black Pixels Versions app .plist bookmarks file into GitHawk?

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.27.2 (1546696737)
Device: iPad6,11 (iOS 12.2)
TestFlight: false
</pre>
</details>

**Comment by BasThomas** [comment]
I'm not sure what you mean. Can you elaborate further? 😇

**Comment by Sherlouk** [comment]
@BasThomas I believe the request here is to allow importing of Bookmarks from external sources such as the [Versions app](https://versionsapp.com/)'s Plist format.

Would be a case of just pulling out the repository URLs, working out which ones are on GitHub and then pulling them into the DB.

**Comment by globdesign** [comment]
> @BasThomas I believe the request here is to allow importing of Bookmarks from external sources such as the [Versions app](https://versionsapp.com/)'s Plist format.
> 
> Would be a case of just pulling out the repository URLs, working out which ones are on GitHub and then pulling them into the DB.

Yes. That was what I had in mind. On macOS, I beleive the file name in question is "com.blackpixel.versions.plist" but I could be wrong - Versions may store the repo bookmark list in some other location in the filesystem.
